changelogs/2023-10-03-changelog.html

```diff
@@ -4,6 +4,10 @@
   <meta charset="utf-8">
   <title>2023-10-03 - v1.0.0 Changelog</title>
 
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
```

Comment:
Are there any benefits to restricting the zoom/scale (minimum-scale and maximum-scale) capabilities?